src/components/light.js

```diff
@@ -20,7 +20,7 @@ module.exports.Component = registerComponent('light', {
     color: {type: 'color', if: {type: ['ambient', 'directional', 'hemisphere', 'point', 'spot']}},
     envMap: {default: '', if: {type: ['probe']}},
     groundColor: {type: 'color', if: {type: ['hemisphere']}},
-    decay: {default: 1, if: {type: ['point', 'spot']}},
+    decay: {default: 2, if: {type: ['point', 'spot']}},
```

Comment:
Why did you change decay?

Reply:
This is @mrxz suggestion in https://github.com/aframevr/aframe/issues/5546#issuecomment-2191818981